### ISSUE TITLE
After Update to 1.5: ticket overviews blank & can't create tickets

### ISSUE THREAD

**Issue by PhoenixZeroDown** [zammad/zammad#982]
Hi there,

after Updating from 1.4 to 1.5 the ticket overviews are blank and I get this error message:

![grafik](https://cloud.githubusercontent.com/assets/24315361/25325094/e4f1f06c-28c9-11e7-8b8b-807cdfa7f37a.png)

![grafik](https://cloud.githubusercontent.com/assets/24315361/25325259/e7bdb85c-28ca-11e7-94a8-e721f5ee12ff.png)

The ticket view is messed up too. I have double entries for Title and Groups and when I want to create a new ticket I get an error message, that 'Group can not be blank'.

![grafik](https://cloud.githubusercontent.com/assets/24315361/25325241/c804d842-28ca-11e7-8e8f-b538873c6b19.png)

### Infos:

* Used Zammad version: 1.5
* Used Zammad installation source: package
* Operating system: Debian Jessie
* Browser + version: Firefox 52.02

**Comment by olivergrahl** [comment]
Did you migrate your database via `rake db:migrate` after upgrading?

**Comment by PhoenixZeroDown** [comment]
No. Just followed the Update Instructions for DEB in the documentation:

shell> sudo systemctl stop zammad

shell> apt-get update
shell> apt-get upgrade

shell> sudo systemctl start zammad

I always did it this way and never had any problems so far

**Comment by monotek** [comment]
Right, the "rake db:migrate" should be done by the updater.
See: https://github.com/zammad/zammad/blob/develop/contrib/packager.io/functions#L191
Any errors wile package update was run?

**Comment by PhoenixZeroDown** [comment]
Here is what I have found in the term.log

# database.yml found. Updating db...
== 20170314000001 FixedTranslation2: migrating ================================
rake aborted!
StandardError: An error has occurred, all later migrations canceled:

Can't load translations from https://i18n.zammad.com/api/v1/translations/en-us: #<Errno::ECONNREFUSED: Failed to open TCP connection to i18n.zammad.com:443 (Connection refused - connect(2) for "i18n.zammad.com" port 443)>

**Comment by monotek** [comment]
OK, seems like a connection error.

@martini 
Why does this happen? 
I thought we put the translations into the package now?
If so the check/download for a more recent version shoul maybe moved to scheduler to prevent erros like this

@PhoenixZeroDown 
Please run:
* zammad run rake db:migrate

**Comment by PhoenixZeroDown** [comment]
Thanks, it works now. What I did:

From cisco firewall I permitted direct internet access for our zammad server, because proxy server was only configured for apt, ran rake db:migrate and everything seems fine now. 

One error message still appears:
![grafik](https://cloud.githubusercontent.com/assets/24315361/25336078/2fba9ee2-28f6-11e7-8636-10afd3255957.png)

**Comment by martini** [comment]
@PhoenixZeroDown 

> One error message still appears:

Is the error message still after restarting Zammad?

**Comment by PhoenixZeroDown** [comment]
Yes. But I shows up only once when the first user logs in. The GeoIP Service is not enabled and this error message only appears on our test server, the productive one is fine.

**Comment by thorsteneckel** [comment]
I hope that this has been resolved. It's been a while. If not please refer to the [community board](http://community.zammad.org/). There are a lot of great people willing to help. Closing now.
